eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -42,6 +44,44 @@ stages:
                         ReleaseSha: $(Build.SourceVersion)
                         RepoId: Azure/azure-sdk-for-python
                         WorkingDirectory: $(System.DefaultWorkingDirectory)
+                        
+          - deployment: PublishDocs
+            displayName: "Docs.MS Release"
+            condition: ne(variables['Skip.PublishDocs'], 'true')
+            environment: githubio
+
+            pool:
+              vmImage: ubuntu-18.04
+
+            strategy:
+              runOnce:
+                deploy:
+                  steps:
+                    - checkout: self
+                    - pwsh: |
+                        Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}
```

Comment:
I know you are just repeating the same pattern here but every time I see this it makes me a little sad. Ideally we would group the artifacts into individual folders in the downloaded artifact but if we cannot do that we should at least have a template that allows us to share this logic because it isn't the easiest code to follow and maintain.